deployment/united-manufacturing-hub/values.yaml

```diff
@@ -1128,30 +1016,24 @@ kafkatoblob:
 redpanda:
   serviceAccount:
     create: false
-#  nameOverride: united-manufacturing-hub-kafka
-  fullnameOverride: united-manufacturing-hub-event-broker
-  external:
-    type: LoadBalancer
+  fullnameOverride: united-manufacturing-hub-kafka
   statefulset:
     replicas: 1
   rbac:
     enabled: true
-  service:
-    type: ClusterIP
-    ports:
-      client: 9092
-      internal: 9093
-      external: 9094
   storage:
     persistentVolume:
-      size: 8Gi
+      size: 10Gi
+  listeners:
+    kafka:
+      port: 9092
   resources:
     cpu:
       cores: 2
```

Comment:
lets remember to move it down to 1 core :)

Reply:
https://github.com/united-manufacturing-hub/united-manufacturing-hub/issues/1554